<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>credit</class>
 <widget class="QWidget" name="credit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>967</width>
    <height>168</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Credit</string>
  </property>
  <widget class="QRadioButton" name="AnnuityButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>96</y>
     <width>109</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Аннуитетный</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="DifferentiatedButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>169</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Дифференцированный</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>11</y>
     <width>98</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>.AppleSystemUIFont</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Сумма кредита</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>43</y>
     <width>87</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>.AppleSystemUIFont</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Срок кредита</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>.AppleSystemUIFont</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Процентная ставка</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>70</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>40</y>
     <width>25</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>.AppleSystemUIFont</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>мес</string>
   </property>
  </widget>
  <widget class="QLabel" name="MonthPay">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>11</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="OverPay">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>68</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="AllPay">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>118</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CalcButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>110</y>
     <width>101</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Рассчитать</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="SummCredit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>125</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="TimeCredit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>125</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="ProcentCredit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>125</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="OverPay_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>68</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Переплата:</string>
   </property>
  </widget>
  <widget class="QLabel" name="MonthPay_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>11</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ежемесячный платеж:</string>
   </property>
  </widget>
  <widget class="QLabel" name="AllPay_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>118</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Общая выплата:</string>
   </property>
  </widget>
  <widget class="QLabel" name="FirstPay">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>10</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Первый платеж:</string>
   </property>
  </widget>
  <widget class="QLabel" name="FirstPay_2">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>10</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="LustPay_">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>70</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Последний платеж:</string>
   </property>
  </widget>
  <widget class="QLabel" name="LustPay">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>70</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
